django-stubs/contrib/admin/options.pyi

```diff
@@ -26,8 +30,11 @@ from django.db.models.fields import Field
 
 IS_POPUP_VAR: str
 TO_FIELD_VAR: str
-HORIZONTAL: Any
-VERTICAL: Any
+HORIZONTAL: Final[Literal[1]] = ...
+VERTICAL: Final[Literal[2]] = ...
+
+# TODO: Is this ok? Is this the way to mark a type as stub-only?
+_Direction = Literal[1, 2]
```

Comment:
I guess `Union[Literal[1], Literal[2]]` is more readable.

Reply:
I'd disagree about the readability, but I'll update accordingly.